src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -504,7 +531,7 @@ function systemStatusGetUpdateStatus() {
 		},
 		dataType: 'html',
 		success: function(data){
-			$('[id^=widget-system_information] #updatestatus').html(data);
+			$('#system-information-widget-updatestatus').html(data);
```

Comment:
@phil-davis - can you test the multiple widget aspect?  (If it needs fixes and they're easily identified, can I ask you as a favour to push them, as I'm not that experienced with pure CSS handling of that kind and it wouldn't be easy for me to be sure if I had fixed them or what is needed).